src/codex_plus/task_api.py

```
@@ -0,0 +1,369 @@
+        **result.metadata
+    }
+    
+    return TaskResult(
+        success=success,
+        output=result.output,
+        error=result.error,
+        metadata=metadata
+    )
+
+
+def Task(subagent_type: str, prompt: str, description: str = None) -> TaskResult:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)